packages/gestalt/src/Badge.css

```diff
@@ -5,6 +5,7 @@
   composes: fontWeightBold from './Typography.css';
   composes: white from './Colors.css';
   font-size: 10px;
+  height: fit-content;
```

Comment:
This is the fix.

Reply:
@dangerismycat can we avoid `height: fit-content`? Firefox doesn't support it yet. https://caniuse.com/mdn-css_properties_height_fit-content

Instead would `align-self` be an option? e.g. `align-self: center;`

Reply:
Good catch! 😅 

`align-self` seems odd in that this isn't actually about the alignment of the component within the flex layout, but the its height. Ideally we get the element to keep its normal height in a flex layout but still allow the Flex to align it. How about `height= min-content`? [That seems to have much better support](https://caniuse.com/?search=height%3A%20min-content):
<img width="949" alt="Screen Shot 2022-02-03 at 5 23 52 PM" src="https://user-images.githubusercontent.com/12059539/152457246-96526521-ef46-4bff-ac75-04726c198247.png">